transformers/struct.go

```diff
@@ -159,6 +160,11 @@ func (t *structTransformer) addColumnFromField(field reflect.StructField, parent
 		IgnoreInTests: t.ignoreInTestsTransformer(field),
 	}
 
+	// Avoid infinite recursion
```

Comment:
I think this comment made more sense before. Now it should be inside the `fieldToJSONSchema` function, if it's necessary.